x-pack/plugins/ingest_pipelines/public/application/sections/pipelines_edit/pipelines_edit.tsx

```diff
@@ -68,88 +68,91 @@ export const PipelinesEdit: React.FunctionComponent<RouteComponentProps<MatchPar
     services.breadcrumbs.setBreadcrumbs('edit');
   }, [services.breadcrumbs]);
 
-  let content: React.ReactNode;
-
   if (isLoading) {
-    content = (
-      <SectionLoading>
-        <FormattedMessage
-          id="xpack.ingestPipelines.edit.loadingPipelinesDescription"
-          defaultMessage="Loading pipeline…"
-        />
-      </SectionLoading>
+    return (
+      <EuiPageContent verticalPosition="center" horizontalPosition="center" color="subdued">
+        <SectionLoading>
+          <FormattedMessage
+            id="xpack.ingestPipelines.edit.loadingPipelinesDescription"
+            defaultMessage="Loading pipeline…"
+          />
+        </SectionLoading>
+      </EuiPageContent>
     );
-  } else if (error) {
-    content = (
-      <>
-        <EuiCallOut
+  }
+
+  if (error) {
+    return (
+      <EuiPageContent
+        verticalPosition="center"
+        horizontalPosition="center"
+        color="danger"
+        data-test-subj="fetchPipelineError"
+      >
+        <EuiEmptyPrompt
+          iconType="alert"
           title={
-            <FormattedMessage
-              id="xpack.ingestPipelines.edit.fetchPipelineError"
-              defaultMessage="Error loading pipeline"
-            />
+            <h2>
+              <FormattedMessage
+                id="xpack.ingestPipelines.edit.fetchPipelineError"
+                defaultMessage="Unable to load existing pipeline"
```

Comment:
Maybe instead of saying "...existing pipeline" we can include the name of the pipeline in the title.